server/Arcadia.Assistant.Web/Startup.cs

```diff
@@ -149,13 +155,25 @@ public void ConfigureContainer(ContainerBuilder builder)
 
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
         // ReSharper disable once UnusedMember.Global
-        public void Configure(IApplicationBuilder app, IHostingEnvironment env, ISecuritySettings securitySettings)
+        public void Configure(
+            IApplicationBuilder app,
+            IHostingEnvironment env,
+            ISecuritySettings securitySettings,
+            IHttpClientFactory httpClientFactory,
+            IDownloadApplicationSettings downloadApplicationSettings)
         {
             if (env.IsDevelopment())
             {
                 app.UseDeveloperExceptionPage();
             }
 
+            this.ActorSystem.ActorOf(
```

Comment:
why is it here it has nothing to do with Configure. Configure must set up all middlewares, it should not start any actors

Reply:
from msdn - The Configure method is used to specify how the app responds to HTTP requests. ConfigureServices is a better place to do it

Reply:
In that method I don't have `IHttpClientFactory` yet. Or I just don't know how to get it.